common/c_cpp/src/c/wombat/wtable.h

```diff
@@ -31,6 +31,8 @@ extern "C" {
 
 typedef void* wtable_t;
 
+/* Indicate the success of wTable functions. */
+#define WTABLE_INSERT_SUCCESS   1
```

Comment:
What @MattMulhern said. It was put in wtable.h so it was accessible to other code using the same. Wtable has weirdness around return codes though, horrible weirdness...